### DISCUSSION TITLE
How to change sampler settings

### DISCUSSION THREAD

**Discussion by prashantstar123** [nuclear-multimessenger-astronomy/nmma#308] [General]
hello All,
Anyone here can tell me how to cahnge sampler settings from dynesty to pymultinest?

**Comment by mcoughlin** [comment]
@tsunhopang maybe you can add a note to the docs on how to list all possible samplers from the command line or something?

**Reply by prashantstar123** [reply]
@mcoughlin hey any update on the above??

**Comment by tsunhopang** [comment]
For the full nmma (i.e., nmma_analysis and nmma_gw_analysis), only `dynesty` is supported. Due to the dimensionality, some common samplers e.g. `pymultinest` will fail.
